PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskJetsEECpbpb.h

```diff
@@ -222,20 +222,40 @@ class AliAnalysisTaskJetsEECpbpb : public AliAnalysisTaskEmcalJet {
     Bool_t fCout; ///<print cout statements 
     Bool_t fisGoodIncEvent; ///<check if event is good 
 
-    Float_t fJet_pt_det; ///< jet pt det 
-    Float_t fJet_pt_tru; ///< jet pt tru
-    Float_t fTrack_pt_det; ///< track pt det 
-    Float_t fTrack_pt_tru; ///< track pt tru
-    Float_t fTrack_eta_det; ///< track eta det 
-    Float_t fTrack_eta_tru; ///< track eta tru 
-    Float_t fTrack_phi_det; ///< track phi det 
-    Float_t fTrack_phi_tru; ///< track phi tru
-    Float_t fTrack_pt_miss; ///< miss track pt 
-    Float_t fTrack_pt_fake; ///< fake track pt 
-    Float_t fTrack_eta_miss; ///< miss track eta 
-    Float_t fTrack_eta_fake; ///< fake track eta  
-    Float_t fTrack_phi_miss; ///< miss track phi  
-    Float_t fTrack_phi_fake; ///< fake track phi
+    // Float_t fJet_pt_det; ///< jet pt det 
+    // Float_t fJet_pt_tru; ///< jet pt tru
+    // Float_t fTrack_pt_det; ///< track pt det 
+    // Float_t fTrack_pt_tru; ///< track pt tru
+    // Float_t fTrack_eta_det; ///< track eta det 
+    // Float_t fTrack_eta_tru; ///< track eta tru 
+    // Float_t fTrack_phi_det; ///< track phi det 
+    // Float_t fTrack_phi_tru; ///< track phi tru
+    // Float_t fTrack_pt_miss; ///< miss track pt 
+    // Float_t fTrack_pt_fake; ///< fake track pt 
+    // Float_t fTrack_eta_miss; ///< miss track eta 
+    // Float_t fTrack_eta_fake; ///< fake track eta  
+    // Float_t fTrack_phi_miss; ///< miss track phi  
+    // Float_t fTrack_phi_fake; ///< fake track phi
+
+    std::vector<float> fJet_ptPhiEta_det; ///< jet pt det 
+    std::vector<float> fJet_ptPhiEta_tru; ///< jet pt tru
+    std::vector<float> fTrack_pt_det; ///< track pt det 
+    std::vector<float> fTrack_pt_tru; ///< track pt tru
+    std::vector<float> fTrack_eta_det; ///< track eta det 
+    std::vector<float> fTrack_eta_tru; ///< track eta tru 
+    std::vector<float> fTrack_phi_det; ///< track phi det 
+    std::vector<float> fTrack_phi_tru; ///< track phi tru
+    std::vector<float> fTrack_pt_miss; ///< miss track pt 
+    std::vector<float> fTrack_pt_fake; ///< fake track pt 
+    std::vector<float> fTrack_eta_miss; ///< miss track eta 
+    std::vector<float> fTrack_eta_fake; ///< fake track eta  
+    std::vector<float> fTrack_phi_miss; ///< miss track phi  
+    std::vector<float> fTrack_phi_fake; ///< fake track phi
+    std::vector<float> fEventTrack_pt; ///< event track pT
+    std::vector<float> fEventTrack_eta; ///< event track eta
+    std::vector<float> fEventTrack_phi; ///< event track phi
+    std::vector<float> fEventTrack_id; ///< event track id
```

Comment:
I don't think you want to stream these? But if you do, you need to increment the ClassDef (in any case, you changed the type of the variables, so you'll need to bump the ClassDef anyway)